src/diffusers/models/unet_2d.py

```diff
@@ -30,7 +30,7 @@ def __init__(
         attention_head_dim=8,
         norm_num_groups=32,
         norm_eps=1e-5,
-    ):
+    ) -> None:
```

Comment:
Nice `None` is the correct return type here according to https://stackoverflow.com/questions/46779046/correct-type-annotation-for-init (no @anton-l ?) 

@sidthekidder, could you maybe also give all the init values some type hints? I've shown one example above :-)

Reply:
@patrickvonplaten yeah, but it has become more of a soft requirement for `__init__`, as it can only return `None`. E.g. in mypy: https://github.com/python/mypy/issues/604
So no worries if we don't annotate them :)

Reply:
Yeah agree here think it's also cleaner to not have a ` -> None` type

Reply:
Makes sense, no return annotation for the constructor is cleaner - removed `None`!